tests/models.py

```diff
@@ -204,3 +201,46 @@ class JoinedEnumB(temporal_sqlalchemy.Clocked, JoinedEnumBase):
     val = sa.Column(sap.ENUM('bar', 'barfoo', name='joined_enum_b_val'))
 
     __mapper_args__ = {'polymorphic_identity': 'enum_b'}
+
+
+class NewStyleModel(Base, temporal_sqlalchemy.TemporalModel):
+    __tablename__ = 'new_style_temporal_model'
+    __table_args__ = {'schema': SCHEMA}
+
+    id = auto_uuid()
+    description = sa.Column(sa.TEXT)
+
+    int_prop = sa.Column(sa.Integer)
+    bool_prop = sa.Column(sa.Boolean)
+    datetime_prop = sa.Column(sa.DateTime(True))
+
+    class Temporal:
+        activity_class = Activity
+        track = ('description', 'int_prop', 'bool_prop', 'datetime_prop')
```

Comment:
Does Paul's suggestion work? Where you use the variable itself instead of the string?